torch/linalg/__init__.py

```diff
@@ -13,3 +13,128 @@
+        -inf   min(sum(abs(x), dim=1))       min(abs(x))
+        0      -- not supported --           sum(x != 0)
+        1      max(sum(abs(x), dim=0))       as below
+        -1     min(sum(abs(x), dim=0))       as below
+        2      2-norm (largest sing. value)  as below
+        -2     smallest singular value       as below
+        other  -- not supported --           sum(abs(x)**ord)**(1./ord)
+        =====  ============================  ==========================
+
+        Default: ``None``
+
+    dim (int, 2-tuple of ints, 2-list of ints, optional): If :attr:`dim` is an int,
+        vector norm will be calculated over the specified dimension. If :attr:`dim`
+        is a 2-tuple of ints, matrix norm will be calculated over the specified
+        dimensions. If :attr:`dim` is None, matrix norm will be calculated
+        when the input tensor has two dimensions, and vector norm will be
+        calculated when the input tensor has one dimension. Default: ``None``
+
+    keepdim (bool, optional): If set to True, the reduced dimensions are retained
+        in the result as dimensions with size one. Default: ``False``
+
+    out (Tensor, optional): The output tensor. Ignored if ``None``. Default: ``None``
+
+    dtype (:class:`torch.dtype`, optional): If specified, the input tensor is cast to
+        :attr:`dtype` before performing the operation, and the returned tensor's type
+        will be :attr:`dtype`. If this argument is used in conjunction with the
+        :attr:`out` argument, the output tensor's type must match this argument;
+        otherwise, a RuntimeError will be raised. This argument is not currently
+        supported for :attr:`ord='nuc'` or :attr:`ord='fro'`. Default: ``None``
+
+Examples::
+
+    >>> import torch
+    >>> from torch import linalg as LA
+    >>> a = torch.arange(9, dtype=torch.float) - 4
+    >>> a
+    tensor([-4., -3., -2., -1.,  0.,  1.,  2.,  3.,  4.])
+    >>> b = a.reshape((3, 3))
+    >>> b
+    tensor([[-4., -3., -2.],
+            [-1.,  0.,  1.],
+            [ 2.,  3.,  4.]])
+
+    >>> LA.norm(a)
+    tensor(7.7460)
+    >>> LA.norm(b)
+    tensor(7.7460)
+    >>> LA.norm(b, 'fro')
+    tensor(7.7460)
+    >>> LA.norm(a, float('inf'))
+    tensor(4.)
+    >>> LA.norm(b, float('inf'))
+    tensor(9.)
+    >>> LA.norm(a, -float('inf'))
+    tensor(0.)
+    >>> LA.norm(b, -float('inf'))
+    tensor(2.)
+
+    >>> LA.norm(a, 1)
+    tensor(20.)
+    >>> LA.norm(b, 1)
+    tensor(7.)
+    >>> LA.norm(a, -1)
+    tensor(0.)
+    >>> LA.norm(b, -1)
+    tensor(6.)
+    >>> LA.norm(a, 2)
+    tensor(7.7460)
+    >>> LA.norm(b, 2)
+    tensor(7.3485)
+
+    >>> LA.norm(a, -2)
+    tensor(0.)
+    >>> LA.norm(b.double(), -2)
+    tensor(1.8570e-16, dtype=torch.float64)
+    >>> LA.norm(a, 3)
+    tensor(5.8480)
+    >>> LA.norm(a, -3)
+    tensor(0.)
+
+Using the :attr:`dim` argument to compute vector norms::
+
+    >>> c = torch.tensor([[1., 2., 3.],
```

Comment:
Do you need to do another `Example::` here and below?

Reply:
It doesn't seem to be necessary, the format of the generated doc looks correct to me.